litellm/router_utils/client_initalization_utils.py

```diff
@@ -186,9 +187,16 @@ def set_client(litellm_router_instance: LitellmRouter, model: dict):
                     f"api_base is required for Azure OpenAI. Set it on your config. Model - {_filtered_model}"
                 )
             azure_ad_token = litellm_params.get("azure_ad_token")
+            azure_ad_token_provider = None
```

Comment:
can we type this correctly, so it's clear what the potential values could be here

Reply:
✅